code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -683,16 +683,25 @@ components:
               accessPoints:
                 $ref: '#/components/schemas/AccessEndpoint'
           minItems: 1
+        kubernetesClusterRef:
+          $ref: '#/components/schemas/KubernetesClusterRef'
         edgeCloudZone:
           $ref: '#/components/schemas/EdgeCloudZone'
 
-    AppZones:
+    AppInstances:
```

Comment:
The name "AppInstances" seems to be conveying the intent that it refers to information about the instances of the application while the object itself and description says it is collection of edge cloud zone where the applications are to be deployed. It seems there is some anomaly here that may be corrected.

Reply:
I agree, I think it may make more sense to call it "AppZones" as it was before.